infra/resources/prod/redis.tf

```
@@ -46,49 +46,49 @@ module "redis_messages" {
 }
 
 
-module "com_redis_messages" {
-  source = "github.com/pagopa/terraform-azurerm-v4//redis_cache?ref=v1.2.1"
+resource "azurerm_redis_cache" "com_redis_messages" {
+  name = "${local.project}-${local.domain}-redis-01"
 
-  name                = "${local.project}-${local.domain}-redis-01"
   resource_group_name = azurerm_resource_group.itn_com.name
   location            = azurerm_resource_group.itn_com.location
 
-  capacity              = 2
-  family                = "C"
-  sku_name              = "Standard"
-  redis_version         = "6"
-  enable_authentication = true
-  zones                 = [1, 2]
+  capacity            = 2
+  family              = "C"
+  sku_name            = "Standard"
+  minimum_tls_version = "1.2"
 
-  // when azure can apply patch?
-  patch_schedules = [{
+  redis_configuration {
+    authentication_enabled = true
+  }
+
+  patch_schedule {
     day_of_week    = "Sunday"
     start_hour_utc = 23
-    },
-    {
-      day_of_week    = "Monday"
-      start_hour_utc = 23
-    },
-    {
-      day_of_week    = "Tuesday"
-      start_hour_utc = 23
-    },
-    {
-      day_of_week    = "Wednesday"
-      start_hour_utc = 23
-    },
-    {
-      day_of_week    = "Thursday"
-      start_hour_utc = 23
-    },
-  ]
+  }
 
-  private_endpoint = {
-    enabled              = true
-    subnet_id            = data.azurerm_subnet.pep.id
-    virtual_network_id   = data.azurerm_virtual_network.vnet_common_itn.id
+  zones = [1, 2]
+
+  tags = local.tags
+}
+
+resource "azurerm_private_endpoint" "this" {
```

`this` can be ambiguous since the scope is the entire `prod` module, use a more descriptive name

```suggestion
resource "azurerm_private_endpoint" "redis_cache_com" {
```